### ISSUE TITLE
Theme broke - fatal error 

### ISSUE THREAD

**Issue by Matrixverse** [siteorigin/settings#55]
Hello,
In the customizer, I edit colors for the mobile menu and I delete all colors and hit the save button and my whole website was down in a second.
I look around my website root files and I find a mistake in wp-content files.

I will share a screenshot so you can investigate further.
If you want to fix this problem just activate any default WP theme and go to customizer and activate mobile menu colors and then reactivate the corp theme and everything will work fine.

Thank you, 
Regards,
VKGteam
![themebrokeamsternanny1](https://user-images.githubusercontent.com/50793707/80695114-b7878300-8ad5-11ea-8266-c906b0c9ea64.png)

**Comment by Matrixverse** [comment]
Hello,Sorry for delaying with reply and I see that you find a way to recreate issue.I see that issue is removed from GitHub so I can't follow further from there?Thank you,Regards30.04.2020. 15:21, Andrew Misplon <notifications@github.com> је написао/ла:

I've recreated the issue, thanks.

To recreate this issue, activate Corp, go to the Navigation section, clear any color value and save.



Notice: Uninitialized string offset: 0 in /wp-content/themes/siteorigin-corp/inc/settings/inc/color.php on line 28



Fatal error: Uncaught Exception: Invalid hex color [] in /wp-content/themes/siteorigin-corp/inc/settings/inc/color.php:28 Stack trace: #0 /wp-content/themes/siteorigin-corp/inc/settings/inc/css_functions.php(55): SiteOrigin_Settings_Color::hex2rgb('') siteorigin/siteorigin-corp#1 /wp-content/themes/siteorigin-corp/inc/settings/settings.php(742): SiteOrigin_Settings_CSS_Functions->rgba(Array) siteorigin/siteorigin-corp#2 [internal function]: SiteOrigin_Settings->css_functions(Array) siteorigin/siteorigin-corp#3 /wp-content/themes/siteorigin-corp/inc/settings/settings.php(656): preg_replace_callback('/\\.([a-z\\-]+) *...', Array, '/* style */ bod...') siteorigin/siteorigin-corp#4 /wp-includes/class-wp-hook.php(287): SiteOrigin_Settings->display_custom_css('') siteorigin/siteorigin-corp#5 /Users/misplon/Sites/siteorigin/wp-includes/class-wp-hook.php(311): WP_Hook->apply_filters(NULL, Array) siteorigin/siteorigin-corp#6 /wp-includes/plugin.php(478): WP_Hook->do_action(Array) siteorigin/siteorigin-corp#7 /wp-content/themes/siteorigin-corp/inc/settings/inc/color.php on line 28

**Comment by Misplon** [comment]
Thanks for your reply and assistance. The issue has been moved to the Settings repository, you're still subscribed and following along. Settings is a submodule of Corp. Once we've resolved here we'll pull the Settings submodule in Corp and then we can release an update.

**Comment by Matrixverse** [comment]
Ok, thanks.After last update of Lite Speed plugin I also experience a lot of problems and I am debugging now but all of them are somehow relate to css or js from page builder. When you plan to release theme and plugin new updates?Thank you.Regards.30.04.2020. 16:24, Andrew Misplon <notifications@github.com> је написао/ла:
Thanks for your reply and assistance. The issue has been moved to the Settings repository, you're still subscribed and following along. Settings is a submodule of Corp. Once we've resolved here we'll pull the Settings submodule in Corp and then we can release an update.
